qa/L0_function_stack/func_stack.log

```diff
@@ -0,0 +1,4 @@
+Traceback (most recent call last):
+  File "./test_pyprof_func_stack.py", line 24, in <module>
+    class TestPyProfFuncStack(unittest.TestCase):
+NameError: name 'unittest' is not defined
```

Comment:
Why is this log catching an error?

Reply:
That must have been an old file.  I deleted it.  You can see it passing successfully in the matching CI branch

Reply:
Yea i saw that which is what prompted the question. Thank you for deleting.